### PR TITLE
dumpsys: add page

### PR DESCRIPTION
**Pull request by marchersimon**
For #4934

### CODE REVIEW

File: pages/android/dumpsys.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- List service-specific arguments for a service:
+
+`dumpsys {{service}} -h`
+
+- Exclude a specific service from the diagnostic output:
+
+`dumpsys --skip {{service}}`
+
+- Specify a timeout period in seconds (defaults to 10s):
```
Comment (marchersimon) [inline pages/android/dumpsys.md:26]: I have no idea what this example does, I just used the description from the Android page.

File: pages/android/dumpsys.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`dumpsys -t {{timeout}}`
```
Comment (marchersimon) [inline pages/android/dumpsys.md:28]: I have no idea, what this example does, I just took the description from the Android page. I hope it's correct.

Comment (bl-ue) [reply pages/android/dumpsys.md:28]: No long option?

Comment (marchersimon) [reply pages/android/dumpsys.md:28]: At least not according to the Android page

Comment (navarroaxel) [reply pages/android/dumpsys.md:28]: I test it but I don't understand what this command does:
```suggestion
- Specify a timeout period in seconds (defaults to 10s):

`dumpsys -t {{seconds}}`
```

**Review by bl-ue — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**